codegen/generators/java/src/main/java/com/whispir/codegen/JavaCustomClientCodegen.java

```diff
@@ -0,0 +1,83 @@
+package com.whispir.codegen;
+
+import java.util.ArrayList;
+import java.util.Arrays;
+import java.util.Comparator;
+import java.util.HashMap;
+import java.util.List;
+import java.util.Map;
+import java.util.stream.Collectors;
+
+import org.openapitools.codegen.*;
+import org.openapitools.codegen.model.*;
+import org.openapitools.codegen.languages.JavaClientCodegen;
+
+public class JavaCustomClientCodegen extends JavaClientCodegen {
+    private Map<String, ModelsMap> modelsMapGlobal = new HashMap<String, ModelsMap>();
+
+    public JavaCustomClientCodegen() {
+        super();
+        apiNameSuffix = "Api";
+    }
+
+    public String getName() {
+        return "java-custom";
+    }
+
+    private class CustomSort implements Comparator<CodegenProperty> {
```

Comment:
Could we rename class to something more descriptive? It's not clear what this sort is doing currently.

`ReadOnlySort` could work well